clustering/infinispan/extension/src/test/resources/org/jboss/as/clustering/infinispan/subsystem/subsystem-infinispan-5_0.xml

```diff
@@ -0,0 +1,148 @@
+                <write-behind modification-queue-size="2048" thread-pool-size="1"/>
+                <property name="location">${java.io.tmpdir}</property>
+            </store>
+            <partition-handling enabled="false"/>
+            <state-transfer timeout="60000" chunk-size="10000"/>
+            <backup-for remote-cache="users" remote-site="LON"/>
+        </replicated-cache>
+        <distributed-cache name="dist" mode="SYNC" l1-lifespan="1200000" owners="4" remote-timeout="35000" segments="2" capacity-factor="1.0" consistent-hash-strategy="INTRA_CACHE" statistics-enabled="true">
+            <locking acquire-timeout="30000" concurrency-level="2000" isolation="READ_COMMITTED" striping="true"/>
+            <transaction mode="FULL_XA" stop-timeout="60000" locking="OPTIMISTIC"/>
+            <eviction max-entries="20000" strategy="UNORDERED"/>
+            <expiration interval="10000" lifespan="10" max-idle="10"/>
+            <mixed-keyed-jdbc-store data-source="ExampleDS" dialect="MYSQL" fetch-state="false" passivation="false" preload="true" purge="false" shared="false" singleton="false">
+                <write-behind modification-queue-size="2048" thread-pool-size="1"/>
+                <binary-keyed-table prefix="ispn_entry" batch-size="100" fetch-size="100">
+                    <id-column name="id" type="VARCHAR"/>
+                    <data-column name="datum" type="BINARY"/>
+                    <timestamp-column name="version" type="BIGINT"/>
+                </binary-keyed-table>
+                <string-keyed-table prefix="ispn_bucket" batch-size="100" fetch-size="100">
+                    <id-column name="id" type="VARCHAR"/>
+                    <data-column name="datum" type="BINARY"/>
+                    <timestamp-column name="version" type="BIGINT"/>
+                </string-keyed-table>
+            </mixed-keyed-jdbc-store>
+            <partition-handling enabled="true"/>
+            <state-transfer timeout="60000" chunk-size="10000"/>
+            <backups>
+                <backup site="NYC" failure-policy="WARN" strategy="SYNC" timeout="12000" enabled="false"/>
+                <backup site="SFO" failure-policy="WARN" strategy="SYNC" timeout="12000" enabled="true"/>
+                <backup site="LON" failure-policy="WARN" strategy="SYNC" timeout="12000" enabled="true">
+                    <take-offline after-failures="3" min-wait="10000"/>
+                </backup>
+            </backups>
+        </distributed-cache>
+    </cache-container>
+    <remote-cache-container connection-timeout="1000"
+                            force-return-values="true"
+                            key-size-estimate="512"
+                            max-retries="2"
+                            name="my-remote-container"
+                            protocol-version="2.5"
+                            server-socket-bindings="jdg1 jdg2 jdg3"
```

Comment:
Since we support failover clusters, the server configuration should be extracted to its own sub-element.
<remote-cache-container>
    <remote-servers>
       <remote-server socket-bindings="jdg1 jdg2 jdg3"/>
       <remote-server cluster-name="failoversite" socket-bindings="jdg4 jdg5 jdg6"/>
    </remote-servers>
</remote-cache-container>

Reply:
My preference is for the schema to support a single <remote-server> initially with the possibility in the future to support multiples.
Probably instead of remote-server (which conveys the concept of a single node) this should be called remote-cluster